.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: 2023 Kavya Shukla <kavyuushukla@gmail.com>
+# X-License-Identifier: GPL-2.0
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0-only
```
Please use the new license identifier. And update other files as well in a separate PR.